simtools/layout/telescope_position.py

```diff
@@ -293,7 +293,10 @@ def _convert(self, crs_from, crs_to, xx, yy):
             raise
         if xx is None or yy is None:
             return np.nan, np.nan
-        _to_x, _to_y = transformer.transform(xx=xx, yy=yy)
+        (
```

Comment:
Is this really what black decided to do with this line? I recommend changing it to 
```
 _to_x, _to_y, = transformer.transform(
     xx=xx,
     yy=yy
)
```

which I assume is what black would do in normal circumstances.

Reply:
Really, for such a short line? I have black as pre-commit hook installed and it didn't change anything in this line. I changed it now to a single line.

Reply:
At some point there was a change here, otherwise it wouldn't have popped up in the review. The original change was very weird so I suggested something more traditional. However, if it fits in one line, that's obviously better.